kagi/settings.py

```diff
@@ -14,9 +14,9 @@
     os.path.join(BASE_DIR, "..", "trusted_attestation_roots"),
 )
 WEBAUTHN_TRUSTED_ATTESTATION_CERT_REQUIRED = getattr(
-    settings, "WEBAUTHN_TRUSTED_ATTESTATION_CERT_REQUIRED", True
+    settings, "WEBAUTHN_TRUSTED_ATTESTATION_CERT_REQUIRED", False
 )
-WEBAUTHN_SELF_ATTESTATION_PERMITTED = getattr(
+WEFBAUTHN_SELF_ATTESTATION_PERMITTED = getattr(
```

Comment:
I think a typo crept in here. (a spurious `F`)  😅

Reply:
Good catch !